unstract/tool-registry/pyproject.toml

```diff
@@ -13,7 +13,7 @@ dependencies = [
     "docker~=6.1.3",
     "jsonschema~=4.18.2",
     "PyYAML~=6.0.1",
-    "unstract-sdk~=0.53.2",
+    "unstract-sdk==0.54.0rc2",
```

Comment:
please recehck the version update

Reply:
Discussed with Hari and hence resolving this.